src/strategies/stablexswap/examples.json

```diff
@@ -0,0 +1,19 @@
+[{
+      "stakingDecimals": 18,
```

Comment:
The format for this file is wrong can you use the same as others like this:
https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/defidollar/examples.json
https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of/examples.json
?

Reply:
Hey @bonustrack - I didn't realize there was a standardized formatting, sorry! I've updated the formatting to match the examples specified and it should be good since calling "npm run test --strategy=stablexswap" works. On a side note when I run npm test locally i have to add an package to use Object.fromEntries. Should I add this package as a dev dependency?

Reply:
Not sure how you get this error, Object.fromEntries should work with dependency